client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -419,6 +422,12 @@ public <K, C> ShuffleReader<K, C> getReaderImpl(
     } else {
       readMetrics = context.taskMetrics().shuffleReadMetrics();
     }
+
+    Configuration readerHadoopConf = RssSparkShuffleUtils.newHadoopConfiguration(sparkConf);
+    for (Map.Entry<String, String> entry : remoteStorage.getConfItems().entrySet()) {
```

Comment:
is `remoteStorage` Initialized in executor side

Reply:
does the pr pass through the 'E2E' tests

Reply:
> is `remoteStorage` Initialized in executor side

No, the actual remoteStorage in executor should  be obtained from the ShuffleHandle.

> does the pr pass through the 'E2E' tests
No, there was a following pr #160  to fix the issue and enhance the e2e system test.